package.json

```diff
@@ -24,9 +24,7 @@
   },
   "files": [
     "src",
-    "dist/standalone.js",
-    "dist/standalone.js.LICENSE.txt",
-    "dist/standalone.js.map",
+    "dist/**/*",
```

Comment:
My bad. I'll push a commit making sure `webpack` cleans the folder before building to avoid publishing random stuff